src/win/fmemopen.c

```diff
@@ -0,0 +1,44 @@
+/* This code was adapted from GitHub: https://github.com/Arryboom/fmemopen_windows */
+
+#include <stdio.h>
```

Comment:
Unfortunately, we can include GPL licensed code.

Reply:
I've re-implemented this function.